test/integration/test_analysis_il.c

```diff
@@ -0,0 +1,58 @@
+// SPDX-FileCopyrightText: 2022 Florian Märkl <info@florianmaerkl.de>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_core.h>
+#include "../unit/minunit.h"
+
+/**
+ * Test running an IL vm with Analysis connection independently of the global user-faced vm
+ */
+static bool test_analysis_il_vm_step() {
+	RzCore *core = rz_core_new();
```

Comment:
dumb question, sorry: is it possible to use the IL VM without Core?

Reply:
Yes, should be possible. You will currently have to set up io manually though to allow it to disassemble.

Reply:
Ok thanks